what is data structure.md

```diff
@@ -0,0 +1,46 @@
+## What is dsa?
+## Classification of Data Structure
+## Popular types of Data Structures:
+
+
+
+# What is dsa?
+
+A data structure is a storage that is used to store and organize data. It is a way of arranging data on a computer so that it can be accessed and updated efficiently.
+
+ data structure is not only used for organizing the data. It is also used for processing, retrieving, and storing data. There are different basic and advanced types of data structures that are used in almost every program or software system that has been developed. So we must have good knowledge about data structures. 
+
+
+# Classification of Data Structure
+
+* Linear data structure: Data structure in which data elements are arranged sequentially or linearly, where each element is attached to its previous and next adjacent elements, is called a linear data structure. 
+
+Examples of linear data structures are array, stack, queue, linked list, etc.
+
+* Static data structure: Static data structure has a fixed memory size. It is easier to access the elements in a static data structure.
+
+An example of this data structure is an array.
+
+* Dynamic data structure: In dynamic data structure, the size is not fixed. It can be randomly updated during the runtime which may be considered efficient concerning the memory (space) complexity of the code. 
+
```

Comment:
Dynamic data structures are designed to give the liberty of change of data structures in the run time.